### PR TITLE
simplify use of 'open' internally

### PR DESCRIPTION
**Pull request by ctb**
See https://github.com/dib-lab/khmer/issues/1577

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make clean diff-cover` If it introduces new functionality, is it tested?
- [x] `make format diff_pylint_report doc` Is it well formatted?
- [x] For substantial changes or changes to the command-line interface, is it
  documented in `CHANGELOG.md`? See [keepachangelog](http://keepachangelog.com/)
  for more details.
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

File: screed/seqparse.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import os
 
 from .createscreed import create_db
-from .openscreed import ScreedDB
+from .openscreed import ScreedDB, Open
```
Comment (betatim) [inline screed/seqparse.py]: Why import `Open` when we use `openscreed.Open` below?

**Comment by betatim** [comment]
LGTM modulo that unused(?) import

File: screed/seqparse.py (hunk 1 above)
Comment (ctb) [inline screed/seqparse.py]: no good reason. Removed in 95a9f55.

**Comment by ctb** [comment]
Reviewed - ready to merge when CI build is done.

**Pull request merged**